match/lib/match/git_helper.rb

```diff
@@ -25,7 +25,7 @@ def self.clone(git_url,
 
       @dir = Dir.mktmpdir
 
-      command = "git clone '#{git_url}' '#{@dir}'"
+      command = %(git clone "#{git_url}" '#{@dir}')
```

Comment:
Can we make this one like the rest with double quotes and escaped double quotes? 😊